bin/sign.py

```diff
@@ -226,6 +229,36 @@ def json_tempfile(path: Path, data: JSON) -> Generator[None, None, None]:
     path.unlink()
 
 
+def parse_env_vars(runtime: str) -> tuple[str, str, str, JSON]:
+    """Parse and return environment variables"""
+    try:
+        aad_id = os.environ["SIGNING_AAD_ID"]
+        tenant_id = os.environ["SIGNING_TENANT_ID"]
+        customer_correlation_id = os.environ["SIGNING_CUSTOMER_CORRELATION_ID"]
+        match runtime:
+            case "win10-x64":
+                # This key code is used for signing .exes and .dlls on both Windows and Mac.

```

Comment:
Just curious, how does this cater to both platforms when you have a platform specific case for Mac right after.

Reply:
It caters to only one platform. For Mac, there is a another case statement.